### PR TITLE
Trying out better description

### PR DESCRIPTION
**Pull request by stalker314314**

### CODE REVIEW

**Comment by stalker314314** [comment]
@matiasdelellis I think we should mention that 4GB of RAM is required. Maybe even slip that additional installation is required, like "Once you install all required software...". If we don't mention that, I think it would be dishonest from out side and it can cause frustration/bad experience for people who install it and cannot use it. What do you think? What else I missed?

**Comment by matiasdelellis** [comment]
About the current description is fine.. :smile: 

> @matiasdelellis I think we should mention that 4GB of RAM is required. Maybe even slip that additional installation is required, like "Once you install all required software...". If we don't mention that, I think it would be dishonest from out side and it can cause frustration/bad experience for people who install it and cannot use it. What do you think? What else I missed?

You're right.. I don't know if say a number (I leave it to your critic), but we must say that It is a service with high requirements, which works progressively, probably slow, and it is essential to read the documentation. It should be clear that just installing it does nothing. :sweat_smile: 

So, don't have to alarm so much, but you have to advice to users. :+1:

**Comment by stalker314314** [comment]
> In case it is not clear, approved ..

It was clear, but I am getting to this repo only in the evenings:)

Please check again! Added warnings

File: appinfo/info.xml [hunk 1]
```diff
@@ -5,6 +5,10 @@
 	<summary>A face recognition app</summary>
 	<description><![CDATA[**Detect and group faces of your loved one in your cloud**
 
+⚠️ This application requires minimum of 12GB of RAM memory to work! See [Requirements](https://github.com/matiasdelellis/facerecognition/wiki/Requirements-and-Limitations) for details.
```
Comment (matiasdelellis) [inline appinfo/info.xml]: 12GB????
I have only 8.. :disappointed:

File: appinfo/info.xml [hunk 2]
```diff
@@ -5,6 +5,10 @@
...
+
+⚠️ Setup of this app requires access to terminal and even getting your hands dirty with installation of additional software. See [Installation](https://github.com/matiasdelellis/facerecognition/wiki/Installation) for details.
```
Comment (matiasdelellis) [inline appinfo/info.xml]: :+1:

File: appinfo/info.xml (hunk 1 above)
Comment (stalker314314) [reply appinfo/info.xml]: OMG, good you noticed:) I didn't copy "-" nicely:)

**Pull request merged**